src/OriginDetection.php

```diff
@@ -164,6 +176,10 @@ private function parseMountInfo($handle)
 
     public function readMountInfo($path)
     {
+        if (!file_exists($path)) {
```

Comment:
File opening can fail for other reasons as well, not to mention this being an poster example of a TOCTOU race. Although the race is likely not a problem for anything under `/proc`, I'm surprised that a scanner didn't flag that anyway.

It seems previous version of the code using `@` and checking returned value was more correct overall.

Reply:
I hear you. I'm not sure why the lints suddenly started flagging `@`...